src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/check_health.py

```diff
@@ -0,0 +1,266 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from .custom import get_docker_command
+from ._docker_utils import _get_aad_token
+from .helm import get_helm_command
+from ._utils import get_registry_by_name
+from ._errors import ErrorClass
+
+
+DOCKER_PULL_SUCCEEDED = "Downloaded newer image for {}"
```

Comment:
does the docker cli get localized?  This won't work for non-English versions if it does.

Reply:
What do you mean by localized? I don't think the docker cli is available in other languages than english

Reply:
Just checked. Docker cli is only available in english, so this should be ok by now :)